code/game/objects/structures/crates_lockers/closets/secure/medical_lockers.dm

```diff
@@ -204,16 +204,19 @@
 	req_access = list(ACCESS_PARAMEDIC)
 
 /obj/structure/closet/secure_closet/paramedic/populate_contents()
+	new /obj/item/radio/headset/headset_med/para(src)
```

Comment:
Do you need one here AND in the garment bag?

Reply:
This will be fixed shortly, woopss